package.json

```diff
@@ -30,12 +30,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/ebaauw/homebridge-rpi.git"
+    "url": "git+https://github.com/wjsroot/homebridge-rpi.git"
```

Comment:
You would need this only if you want to publish an `npm` package under your own name.  In that case, you would have to change the package name as well, e.g. to `@wjsroot/homebridge-rpi`, as the `npm` repository requires package names to be unique.